### PR TITLE
Update authorize-ios docs

### PR DESCRIPTION
**Pull request by imurchie**
## Proposed changes

Make sure the documentation of `authorize-ios` points to the right place.
## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules
## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by bootstraponline** [comment]
:+1:

File: docs/en/appium-setup/running-on-osx.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ Appium on OS X supports iOS and Android testing.
 * Make sure you have Xcode and the iOS SDK(s) installed. Xcode version 7.1 is
   recommended as earlier versions of Xcode are limited in which versions of iOS
   they can test against. See the next section for more detail.
-* You need to authorize use of the iOS Simulator by running `sudo authorize_ios`
-  (`authorize_ios` is a binary made available by the Appium npm package). If you
+* You need to authorize use of the iOS Simulator by running `sudo node_modules/.bin/authorize-ios`
```
Comment (jlipps) [inline docs/en/appium-setup/running-on-osx.md]: this isn't valid if someone had run `npm install -g appium`, right? I thought our canonical way of doing this was to also install the authorize-ios package?

Comment (jlipps) [inline docs/en/appium-setup/running-on-osx.md]: also, i wonder if `$(npm bin)/authorize-ios` is more robust than `node_modules/.bin/authorize-ios`

Comment (imurchie) [inline docs/en/appium-setup/running-on-osx.md]: `$(npm bin) won't work either for global install. I'm fine with just saying install globally, then run `authorize-ios`.

Comment (imurchie) [inline docs/en/appium-setup/running-on-osx.md]: I think, actually, that if you install globally it is available already through `authorize-ios`.

**Comment by imurchie** [comment]
Added some info, based on the comments of @jlipps.

**Comment by jlipps** [comment]
:+1:

**Pull request merged**